theia-example-updater/src/electron-browser/updater/theia-updater-preferences.ts

```diff
@@ -0,0 +1,28 @@
+/********************************************************************************
+ * Copyright (C) 2020 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { PreferenceSchema } from '@theia/core/lib/common/preferences/preference-schema';
+
+export const theiaUpdaterPreferenceSchema: PreferenceSchema = {
+    'type': 'object',
+    'properties': {
+        'theiaupdater.reportOnStart': {
+            type: 'boolean',
+            description: 'Whether Theia will report available updates after application start.',
```

Comment:
```suggestion
            description: 'Report available updates after application start.',
```
I'd change it to this because there settings show a checkbox left next to this. This IMO makes "Whether Theia will" unnecessary.

Reply:
done